{
    "BackupId": 92500,
    "BackupConfigurationId": 101145,
    "BackupConfigurationName": "Database Server Backup",
    "BackupConfigurationIsDeleted": false,
    "ComputerName": "Database Server",
    "MachineAgentId": 95874,
    "State": "CompletedWithErrors",
    "CanRestore": true,
    "StartTime": "\/Date(1351118760000)\/",
    "CompletedTime": "\/Date(1351119033000)\/",
    "Duration": "00:04:33",
    "FilesSearched": "11936",
    "BytesSearched": "3.7 GB",
    "FilesBackedUp": "6",
    "BytesBackedUp": "178.1 MB",
    "NumErrors": 2,
    "Reason": "Success",
    "Diagnostics": "Completed With Errors",
    "ErrorList": [
        {
            "BackupReportID": 48785,
            "ListIndex": 3210,
            "ErrorType": 101,
            "ErrorDesc": "The agent experienced a problem. (Rax)",
            "ExceptionDesc": "Cannot open file \"C:\\Program Files\\Microsoft
                  SQL Server\\MSSQL10_50.MSSQLSERVER\\MSSQL\\DATA\\tempdb.mdf\
                  ". The process cannot access the file because it is being 
                  used by another process. .",
            "ExceptionDetails": "1: [WindowsFs.cpp: 157-rax::WindowsFs::Open]
                  Error Code(313): Cannot open file \"C:\\Program Files\\
                  Microsoft SQL Server\\MSSQL10_50.MSSQLSERVER\\MSSQL\\
                  DATA\\tempdb.mdf\". The process cannot access the file 
                  because it is being used by another process. .",
            "ExceptionCode": 313,
            "Path": "C:\\Program Files\\Microsoft SQL Server\\MSSQL10_50.
                  MSSQLSERVER\\MSSQL\\DATA\\tempdb.mdf"
        },
        {
            "BackupReportID": 48785,
            "ListIndex": 3211,
            "ErrorType": 101,
            "ErrorDesc": "The agent experienced a problem. (Rax)",
            "ExceptionDesc": "Cannot open file \"C:\\Program Files\\Microsoft
                  SQL Server\\MSSQL10_50.MSSQLSERVER\\MSSQL\\DATA\\templog.ldf
                  \". The process cannot access the file because it is being 
                  used by another process. .",
            "ExceptionDetails": "1: [WindowsFs.cpp: 157-rax::WindowsFs::Open]
                  Error Code(313): Cannot open file \"C:\\Program Files\
                  \Microsoft SQL Server\\MSSQL10_50.MSSQLSERVER\\MSSQL\\
                  DATA\\templog.ldf\". The process cannot access the file 
                  because it is being used by another process. .",
            "ExceptionCode": 313,
            "Path": "C:\\Program Files\\Microsoft SQL Server\\MSSQL10_50.
                  MSSQLSERVER\\MSSQL\\DATA\\templog.ldf"
        }
    ]
}      
